azure-pipelines.yml

```
@@ -200,6 +206,7 @@ jobs:
 
 
   - script: |
+      set /p ARCH=$(Platform)
```

```suggestion
      set ARCH=$(Platform)
```

`/p` means get value from user input, thus this env is missing.

---

BTW, L210 works because it uses `<` to read from a file.